src/handler.cpp

```diff
@@ -1063,6 +1063,8 @@ void affect_total(struct char_data * ch)
     // Ranged weapons grant 0 reach if pistol, 2 reach if anything else with a bayonet, 1 reach for all other categories.
     else {
       if (GET_WEAPON_SKILL(weapon) != SKILL_PISTOLS)
```

Comment:
Chaining the check like this technically works, but it's a lot cleaner to write `if (GWS != pistols && GWS != tasers &&...)`. More readable too.